packages/gatsby-plugin-gatsby-cloud/src/models/components/index.ts

```diff
@@ -0,0 +1,5 @@
+import IBaseButtonProps from "./base-button-props"
```

Comment:
If you're adding TypeScript files you'll need to adjust the toolchain / build command. I think you can remove the TypeScript files?

Reply:
I need them for the enum

Reply:
But I don't see them used anywhere? They'll also not be built as babel/tsc is not converting them